src/System.Security.Cryptography.Algorithms/tests/HmacSha1Tests.cs

```diff
@@ -39,6 +39,22 @@ protected override HashAlgorithm CreateHashAlgorithm()
         protected override int BlockSize { get { return 64; } }
 
         [Fact]
+        public void HmacSha1_Byte_Constructors()
```

Comment:
Can you add a test with a known vector to verify we're plumbing these correctly?